sourcegraph/about
# Host use case guides

Closes #5583, hosts use case guides

### Changelog
- Adds new use case guides & updates meta titles according to meta

### Test
1. Ensure prettier has standardized the proposed changes.
2. Navigate to `/guides/code-health-overview.pdf` & confirm meta & pdf copy
3. Navigate to `/guides/code-reuse-overview.pdf` & confirm meta & pdf copy
4. Navigate to `/guides/code-security-overview.pdf` & confirm meta & pdf copy
5. Navigate to `/guides/incident-response-overview.pdf` & confirm meta & pdf copy
6. Navigate to `/guides/onboarding-overview.pdf` & confirm meta & pdf copy
